var/spack/repos/builtin/packages/z-checker/package.py

```diff
@@ -30,3 +31,9 @@ def configure_args(self):
         else:
             args += ['--disable-mpi']
         return args
+ 
+    def url_for_version(self, version):
+        """provide url to ensure that download counting via github releases
+        works accurately"""
+        url = "https://github.com/CODARcode/Z-checker/releases/download/{0}/Z-checker-{0}.tar.gz"
+        return url.format(version)
```

Comment:
Why is this needed? Shouldn't it already download from this URL?

Reply:
@disheng222 If I recall, if you use the url https://github.com/szcompressor/SZ/releases/download/v2.1.11/SZ-2.1.11.tar.gz you don't get the release tracking information.  Is this correct?